react/components/Select/ClearIndicator.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import ClearIcon from '../icon/Clear'
+
+const ClearIndicator = ({ innerProps }) => {
+  return (
+    <div
+      className="flex items-center h-100 pr4 pointer silver hover-gray"
```

Comment:
Não usar as classes `silver` e `hover-gray`, elas estão deprecated.

Usar os color tokens como `c-muted-NUMERO` e `hover-c-muted-NUMERO`

Reply:
`hover-c-muted-NUMERO` doesn't exist, should I open a PR on `vtex-tachyons`?